### PR TITLE
[New arch] Stop supporting <10 servers

### PR DESCRIPTION
**Pull request by davigonz**
Complete https://github.com/owncloud/android/issues/2636
Needed for https://github.com/owncloud/android/pull/2648

### CODE REVIEW

File: owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/files/FileUtils.java [hunk 1]
```diff
@@ -45,41 +45,15 @@ public static String getParentPath(String remotePath) {
...
-     */
-    public static boolean isValidPath(String path, boolean versionSupportsForbidenChars) {
-        boolean result = true;
-
-        Log_OC.d(TAG, "path ....... " + path);
-        if (!versionSupportsForbidenChars &&
-                (path.contains("\\") || path.contains("<") || path.contains(">") ||
-                        path.contains(":") || path.contains("\"") || path.contains("|") ||
-                        path.contains("?") || path.contains("*"))) {
+        if (fileName.contains(PATH_SEPARATOR)) {
```
Comment (abelgardep) [inline owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/files/FileUtils.java]: Names with "\" or just an "*" are not allowed neither. Maybe we should check which characters are forbidden. What do you think, @davigonz ?

Comment (davigonz) [reply owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/files/FileUtils.java]: Checking this...

**Comment by davigonz** [comment]
@abelgardep done, could you approve it if it looks good to you?

**Comment by abelgardep** [comment]
Will be merged in https://github.com/owncloud/android-library/pull/286

**Pull request closed without merging**